scripts/create_pedestal_file.py

```diff
@@ -1,21 +1,22 @@
 import sys
 sys.path.insert(0,'/home/pawel1/Pulpit/Astrophysics/CTA/soft/ctapipe_io_lst')
+sys.path.insert(0, '/home/pawel1/Pulpit/Astrophysics/CTA/cta-lstchain')
```

Comment:
Can we avoid this kind of import that depends on a specific path and prevent other users to use the code?
Why would we need such an import anyway?

Reply:
Yes, sorry my mistake. We don't need it.  I removed.

Reply:
Thank you.
If you can have a look to my other comments we could merge this.